phys2bids/tests/test_integration.py

```diff
@@ -0,0 +1,466 @@
+                      log_info) if 'Timepoints expected' in log_str]
+    timepoints_found = log_info[timepoints_idx[0]]
+    assert '158' in timepoints_found
+
+    # Check sampling frequency
+    sampling_idx = [log_idx for log_idx, log_str in enumerate(
+                      log_info) if 'Sampling Frequency' in log_str]
+    sampling_found = log_info[sampling_idx[0]]
+    assert '1000.0' in sampling_found
+
+    # Check sampling frequency
+    started_idx = [log_idx for log_idx, log_str in enumerate(
+                      log_info) if 'Sampling started' in log_str]
+    started_found = log_info[started_idx[0]]
+    assert '0.24499999999989086' in started_found
+
+    # Check start time
+    start_idx = [log_idx for log_idx, log_str in enumerate(
+                      log_info) if 'first trigger' in log_str]
+    start_found = log_info[start_idx[0]]
+    assert 'Time 0' in start_found
+
+    # Checks json file
+    # json_data = json.load(os.path.join(test_path, 'tutorial_file.json'))
+    with open(os.path.join(test_path, 'tutorial_file.json')) as json_file:
+        json_data = json.load(json_file)
+
+    # Compares values in json file with ground truth
+    assert json_data['SamplingFrequency'] == 1000.0
+    assert json_data['StartTime'] == 0.24499999999989086
+    assert json_data['Columns'] == ['time', 'Trigger', 'CO2', 'O2', 'Pulse']
+
+    # Remove generated files
+    for filename in glob.glob(os.path.join(test_path, 'phys2bids*')):
+        os.remove(filename)
+    # for filename in glob.glob(os.path.join(test_path, 'tutorial*')):
+    #     os.remove(filename)
+
+
+def test_integration_acq():
+    """
+    Does the integration test for an acq file
+    """
+
+    url = 'https://osf.io/27gqb/download'
+    test_path = resource_filename('phys2bids', 'tests/data')
+    test_filename = 'Test_belt_pulse_samefreq.acq'
+    test_full_path = os.path.join(test_path, test_filename)
+    wget.download(url, test_full_path)
+    test_chtrig = 3
+
+    phys2bids(filename=test_filename, indir=test_path, outdir=test_path,
+              chtrig=test_chtrig, num_timepoints_expected=1)
+
+    # Check that files are generated
+    assert os.path.isfile(os.path.join(test_path, 'Test_belt_pulse_samefreq.log'))
+    assert os.path.isfile(os.path.join(test_path, 'Test_belt_pulse_samefreq.json'))
+    assert os.path.isfile(os.path.join(test_path, 'Test_belt_pulse_samefreq.tsv.gz'))
+    assert os.path.isfile(os.path.join(test_path, 'Test_belt_pulse_samefreq_trigger_time.png'))
+
+    # Read log file (note that this file is not the logger file)
+    with open(os.path.join(test_path, 'Test_belt_pulse_samefreq.log')) as log_info:
+        log_info = log_info.readlines()
+
+    # Check timepoints expected
+    expected_idx = [log_idx for log_idx, log_str in enumerate(
+                      log_info) if 'Timepoints expected' in log_str]
+    expected_found = log_info[expected_idx[0]]
+    assert '1' in expected_found
```

Comment:
As in the above test, it might be worth making this (and the below checks) a for-loop, or creating that helper function or something!